idb/postgres/postgres_rand_test.go

```diff
@@ -275,53 +275,19 @@ func TestWriteReadResources(t *testing.T) {
 	require.NoError(t, err)
 	defer tx.Rollback(context.Background())
 
-	/*
-		l, err := ledgerforevaluator.MakeLedgerForEvaluator(tx, basics.Round(0))
-		require.NoError(t, err)
-		defer l.Close()
-
-		ret, err := l.LookupResources(resources)
-		require.NoError(t, err)
-	*/
-
-	for address, creatables := range resources {
+	for address, datum := range data {
```

Comment:
In the original there was a nested loop here. Is that missing because the test only creates (and datum only has) up to one of each creatable compared to the original data type that used a slice?

Reply:
There was only one of each kind so a for-loop was unnecessary. However, there was an unused constant `const numCreatables = 20` inside `generateAccountData()` which I've removed. Perhaps the original intent was to create 20 random creatables. In that case, the for loop would have made perfect sense.

Reply:
[Here](https://github.com/winder/indexer/blob/develop/idb/postgres/postgres_rand_test.go#L236-L242) are the original 2 creatables being added to the map (rather than slice) which is then being iterated over in the original test.